src/java.base/share/classes/java/lang/foreign/ValueLayout.java

```diff
@@ -105,6 +104,9 @@ public sealed interface ValueLayout extends MemoryLayout permits
      * The returned var handle checks that accesses are aligned according to this value layout's
      * {@linkplain MemoryLayout#byteAlignment() alignment constraint}.
      *
+     * @apiNote This method is similar, but more efficient, than calling {@code MemoryLayout#varHandle(PathElement...)}
+     * with zero arguments, as it avoids the creation of the var args array.
```

Comment:
perhaps we should say "with an empty path element array" otherwise one might confuse it with this method?